src/Digdir.Domain.Dialogporten.WebApi/appsettings.Development.json

```
@@ -6,6 +6,10 @@
     }
   },
   "Infrastructure": {
+    "Redis":{
+      "Enabled": false,
+      "ConnectionString": "TODO: Add to local secrets"
```

```suggestion
      "ConnectionString": "redis://localhost:6379"
```

For local kan denne hardkodes til den som kjører i docker compose?

---

hm, ja sant!

---

Blir isåfall `redis://dialogporten-redis:6379` isåfall

---

Hm, nah .. ?
Det blir localhost om vi kjører APIet i IDE og resten i docker compose. De nås på localhost når context er den faktiske docker-hosten (inne i hver container må de navna der brukes, da er "localhost" containeren selv så vidt jeg har forstått)

På samme måte du kobler til localhost for å nå DB i PgAdmin

---

> Det blir localhost om vi kjører APIet i IDE og resten i docker compose.
Yeah, sant sant! Godt poeng